app/src/main/java/pm/gnosis/heimdall/ui/transactions/view/status/TransactionStatusViewModel.kt

```diff
@@ -21,12 +20,8 @@ class TransactionStatusViewModel @Inject constructor(
     override fun observeUpdates(id: String): Observable<ViewUpdate> =
         infoRepository.loadTransactionInfo(id)
             .flatMap { info ->
-                if (info.gasToken == ERC20Token.ETHER_TOKEN.address) {
-                    Single.just(ERC20Token.ETHER_TOKEN).map { info to it }
-                } else {
-                    tokenRepository.loadToken(info.gasToken).map { info to it }
-                        .onErrorReturnItem(info to ERC20Token(info.gasToken, decimals = 0))
-                }
+                tokenRepository.loadToken(info.gasToken).map { info to it }
+                    .onErrorReturnItem(info to ERC20Token(info.gasToken, decimals = 0))
```

Comment:
`decimals = 0`?

Reply:
Error fallback. If it is an unknown token we assume decimals 0